### PR TITLE
Only allow send bonuses to otech user

### LINKED ISSUE #834: Only allow `send bonuses` to otech user

### PR DESCRIPTION
**Pull request by Ovifer13**
**Issue #834**

- [x] Fix advanced wallet configuration
- [x] Allow only otech to send bonuses

### CODE REVIEW

**Comment by Ovifer13** [comment]
@otech47 test this when you can, go to any project and see if you have the option to send bonuses. 
I tested with my user but I want to make sure it works with yours.

**Comment by sofiaromorales** [comment]
> @otech47 test this when you can, go to any project and see if you have the option to send bonuses. I tested with my user but I want to make sure it works with yours.

It works fine 👍

File: srcv2/components/AdvancedWalletSetup.js [hunk 1]
```diff
@@ -116,6 +114,7 @@ const AdvancedWalletSetup = () => {
                         type='text'
                         placeholder={r.label}
                         value={r.value}
+                        onChange={(e) => { r.setValue(e.target.value) }}
```
Comment (sofiaromorales) [inline srcv2/components/AdvancedWalletSetup.js:117]: @Ovifer13 why this change?

Comment (Ovifer13) [reply srcv2/components/AdvancedWalletSetup.js:117]: When I did this I forgot to add the onChange value to the input, the input wasn't doing anything till I added this. That's why Oscar was getting that error the other day in the meeting that the button wasn't getting enabled.

File: srcv2/pages/ProjectDetailPage.js [hunk 2]
```diff
@@ -181,7 +181,7 @@ const ProjectDetailPage = (props) => {
                             Active Contributors
                         </p>
                     </div>
-                    {sessionUser().wallet && sessionUser().wallet.invoice_macaroon &&
+                    {sessionUser().wallet && sessionUser().wallet.invoice_macaroon && sessionUser().github_handle.split('/')[3] == 'otech47' &&
```
Comment (otech47) [inline srcv2/pages/ProjectDetailPage.js]: use the contributor ID here as an environment variable `TEMP_AUTHORIZED_SUPERUSER_ID`

**Review by otech47 — CHANGES_REQUESTED** [review]
we need to enforce this on the backend otherwise the API is still vulnerable to an attack that can drain all funds from the LNBank wallet

**Review by otech47 — APPROVED** [review]
